server/accounts.go

```diff
@@ -1338,19 +1338,19 @@ func (a *Account) sendTrackingLatency(si *serviceImport, responder *client) bool
 	// FIXME(dlc) - We need to clean these up but this should happen
 	// already with the auto-expire logic.
 	if responder != nil && responder.kind != CLIENT {
-		a.mu.Lock()
+		si.acc.mu.Lock()
```

Comment:
Should this be cherry picked to main?

Reply:
It is already like that in main. I had it changed to `a.mu.Lock` in the "route pool and per account" PR because I got a race. But since then, you have fixed what was actually causing the race. See comment: https://github.com/nats-io/nats-server/pull/4115/commits/8d2683a062c56343b4de91d04027c1f168cb1197